client/components/search-themes/index.tsx

```diff
@@ -45,6 +45,10 @@ const SearchThemes: React.FC< SearchThemesProps > = ( { query, onSearch } ) => {
 	};
 
 	const updateInput = ( updatedInput: string ) => {
+		// Clean up duplicate criteria
```

Comment:
I feel like we should not put this logic here. This method should be just updating the search box. Could we add the logic in the `suggest()` function instead? What do you think?

Reply:
+1, seems that there's already clean-up code in the [suggest](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/search-themes/index.tsx#L69) function.

Reply:
Moved to `suggest`method